Framework/PythonInterface/inc/MantidPythonInterface/kernel/Converters/ContainerDtype.h

```diff
@@ -0,0 +1,73 @@
+#ifndef MANTID_PYTHONINTERFACE_CONVERTERS_CONTAINERDTYPE_H_
+#define MANTID_PYTHONINTERFACE_CONVERTERS_CONTAINERDTYPE_H_
+
```

Comment:
Thanks for moving the files and cleaning up the headers.
It is generally good practice to include what you use in a file so that users of it don't have to worry about what they need to include to get it working. As the implementation uses `std::string` and type_traits I think we still need to include `<string>` and `<type_traits>`?

Reply:
No problem! Ah yes I see - I'll add those in just now and push the changes.

Reply:
Thanks!